src/api/modules/workspace/create.ts

```diff
@@ -103,6 +103,16 @@ export const workspaceCreator: WorkspaceCreator = async ({
 
     return 'created'
   } catch (err) {
+    if (err.response?.data.code === 'ArgumentNull') {
+      throw createFlowIssueError(
+        `You need to pick a name for the new workspace. Run ${chalk.green(
+          'vtex workspace create [WORKSPACENAME]'
+        )}. You can name it using your name with a number, for example ${chalk.green(
+          `vtex workspace create johndoe01`
+        )}`
```

Comment:
What about adding it to the messages file?

Reply:
And using the colors from ColorifyConstants

Reply:
Good idea